testnets/holesky/commit-boost/cb.docker-compose.yml

```diff
@@ -1,54 +1,87 @@
 services:
   cb_bolt_sidecar:
-    image: ghcr.io/chainbound/bolt-sidecar:v0.3.1-alpha
+    image: ghcr.io/chainbound/bolt-sidecar:v0.4.0-alpha
     container_name: cb_bolt_sidecar
-    env_file:
-      - bolt-sidecar.env
     environment:
-      CB_MODULE_ID: BOLT
-      CB_CONFIG: /cb-bolt-config.toml
-    ports:
-      - 8017:8017
-      - 18550:18550
-    networks:
-      - monitoring_network
+      CB_MODULE_ID: BOLT_SIDECAR
+      CB_CONFIG: /cb-config.toml
+      CB_SIGNER_JWT: ${CB_JWT_BOLT_SIDECAR}
+      CB_SIGNER_URL: http://cb_signer:20000
+      CB_METRICS_PORT: 10000
+      CB_LOGS_DIR: /var/logs/commit-boost
     volumes:
+      - ./cb-bolt-config.toml:/cb-config.toml:ro
+      - ./logs/bolt_sidecar:/var/logs/commit-boost
       - ./delegations.json:/etc/delegations.json:ro
+    networks:
+      - signer_network
+      - monitoring_network
+    depends_on:
+      cb_signer:
+        condition: service_healthy
   cb_pbs:
-    image: ghcr.io/chainbound/bolt-boost:v0.3.0-alpha
+    healthcheck:
+      test: curl -f http://localhost:18551/eth/v1/builder/status
+      interval: 30s
+      timeout: 5s
+      retries: 3
+      start_period: 5s
+    image: ghcr.io/chainbound/bolt-boost:v0.4.0-alpha
     container_name: cb_pbs
     ports:
-      - 18551:18551
+      - 0.0.0.0:18551:18551
     environment:
-      CB_CONFIG: /cb-bolt-config.toml
+      CB_CONFIG: /cb-config.toml
       CB_METRICS_PORT: 10000
       CB_LOGS_DIR: /var/logs/commit-boost
+      CB_PBS_ENDPOINT: 0.0.0.0:18551
     volumes:
-      - ./cb-bolt-config.toml:/cb-bolt-config.toml:ro
+      - ./cb-bolt-config.toml:/cb-config.toml:ro
       - ./logs/pbs:/var/logs/commit-boost
     networks:
       - monitoring_network
+  cb_signer:
+    healthcheck:
+      test: curl -f http://localhost:20000/status
+      interval: 30s
+      timeout: 5s
+      retries: 3
+      start_period: 5s
+    image: ghcr.io/commit-boost/signer:latest
+    container_name: cb_signer
+    environment:
+      CB_CONFIG: /cb-config.toml
+      CB_JWTS: ${CB_JWTS}
+      CB_SIGNER_PORT: 20000
+      CB_METRICS_PORT: 10000
+      CB_LOGS_DIR: /var/logs/commit-boost
+      CB_SIGNER_LOADER_FILE: /keys.json
+    volumes:
+      - ./cb-bolt-config.toml:/cb-config.toml:ro
+      - ./keys.json:/keys.json:ro
+      - ./logs/signer:/var/logs/commit-boost
+    networks:
+      - signer_network
+      - monitoring_network
   cb_prometheus:
-    image: prom/prometheus:latest
+    image: prom/prometheus:v3.0.0
     container_name: cb_prometheus
     ports:
-      - 9090:9090
+      - 127.0.0.1:9090:9090
```

Comment:
Are we sure here 127.0.0.1 is okay and we don't need 0.0.0.0? cc @mempirate same below for grafana